todo/src/actions/index.js

```diff
@@ -0,0 +1,26 @@
+export const ADD_TODO = 'ADD_TODO';
+export const UPDATE_COMPLETED = 'UPDATE_COMPLETED';
+export const DELETE_ITEM = 'DELETE_ITEM';
+
+let nextID = 0;
```

Comment:
I would recommend handling where you're pulling the id from to be somewhere else.  You can handle that logic in the addTodo Handler